docs/index.md

```diff
@@ -3,174 +3,270 @@
+```
 
-```
\ No newline at end of file
+You can also use specific commit images just substituting `latest` for the desired commit SHA.
+## Variables for OCM Terraform Provider
+
+The following variables that you use to create a ROSA cluster with the Terraform provider.
+
+### Required
+
+- `aws_account_id` (String) Identifier of the AWS account.
+- `cloud_region` (String) Cloud region identifier, for example 'us-east-1'.
+- `name` (String) Name of the cluster. Must be a maximum of 15 characters in length.
+
+### Optional
+
+- `autoscaling_enabled` (Boolean) Enables autoscaling.
+- `availability_zones` (List of String) availability zones
+- `aws_private_link` (Boolean) Provides private connectivity between VPCs, AWS services, and your on-premises networks, without exposing your traffic to the public internet.
+- `aws_subnet_ids` (List of String) aws subnet ids
+- `channel_group` (String) Name of the channel group from which to select the OpenShift cluster version, for example 'stable'.
+- `client_id` (String) OpenID client identifier.
+- `client_secret` (String, Sensitive) OpenID client secret.
+- `compute_machine_type` (String) Identifier of the machine type used by the compute nodes, for example `r5.xlarge`. Use the `ocm_machine_types` data source to find the possible values.
+- `default_mp_labels` (Map of String) Labels for the default machine pool. Format should be a comma-separated list of '{"key1"="value1", "key2"="value2"}'. This list will overwrite any modifications made to Node labels on an ongoing basis.
+- `destroy_timeout` (Number) Timeout in minutes for addressing cluster state in destroy resource. Default value is 60 minutes.
+- `disable_scp_checks` (Boolean) Enables you to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics.
+- `disable_waiting_in_destroy` (Boolean) Disable addressing cluster state in the destroy resource. Default value is false
+- `disable_workload_monitoring` (Boolean) Enables you to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics.
+- `ec2_metadata_http_tokens` (String) Which ec2 metadata mode to use for metadata service interaction options for EC2 instancescan be optional or required, available only from 4.11.0
+- `etcd_encryption` (Boolean) Encrypt etcd data.
+- `external_id` (String) Unique external identifier of the cluster.
+- `fips` (Boolean) Create cluster that uses FIPS Validated / Modules in Process cryptographic libraries
+- `host_prefix` (Number) Length of the prefix of the subnet assigned to each node.
+- `insecure` (Boolean) When set to 'true' enables insecure communication with the server. This disables verification of TLS certificates and host names and it isn't recommended for production environments.
```

Comment:
Add a comma to the second sentence between the two clauses and break up the contraction.